src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec

```diff
@@ -51,7 +51,9 @@
     </contentFiles>
   </metadata>
   <files>
+    <!-- Include PDB unless embedded into DLL -->
     <file src="Dlls\MSBuildTask\Microsoft.Build.Tasks.CodeAnalysis.dll" target="lib\netstandard1.3" />
+    <file src="Dlls\MSBuildTask\Microsoft.Build.Tasks.CodeAnalysis.pdb*" target="lib\netstandard1.3" />
```

Comment:
why do we need the `*` here?

Reply:
To make build work locally and on Jenkins, where the PDBs are embedded.

Reply:
Gotcha. Using a glob to allow it to be excluded without error.

Reply:
Yup.

Reply:
@tmat @jaredpar Can we add that explanation in a comment please? :smile:

Reply:
There is a comment.